aries-site/src/pages/foundation/date-and-time.mdx

```diff
@@ -0,0 +1,272 @@
+    bestPractice={{
+      type: 'dont',
+      message: `Don't include additional information if it isn't needed. This will hurt readability and cause issues with layouts at smaller breakpoints.`,
+    }}
+  >
+    <RelativeTimeExample bestPractice={false} />
+  </Example>
+</BestPracticeGroup>
+
+## Abbreviating date and time
+
+When space in the UI is limited, units of time, days of the week, and months can be abbreviated.
+
+Some contexts in which abbreviations may be needed are:
+
+- Within cells of a data table
+- At smaller breakpoints
+
+| Units of time  | Abbreviation |
+| -------------- | ------------ |
+| millisecond(s) | ms           |
+| second(s)      | sec          |
+| minute(s)      | min          |
+| hour(s)        | hr           |
+| day(s)         | day, days    |
+| week(s)        | wk           |
+| month(s)       | mo           |
+| year(s)        | yr           |
+
+To abbreviate days of the week or months, use the [Javascript Intl.DateTimeFormat method](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat). The general convention for English is to use the first 3 letters of the day or month name.
+
+<Example
+  code="https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/foundation/date-and-time/AbbreviatedDateExample.js"
+  horizontalLayout
+  height="none"
+>
+  <AbbreviatedDateExample />
+</Example>
+
+## Zero-padding
+
+When a user needs to compare time stamps such as in a list or column, zero-padding ensures units are uniform and aligned for easy readability and comparison. When applicable, a leading zero should be included within both the date and time.
+
+Zero-padding is not needed when context contains only one time stamp, see [notification example](#notification).
+
+<BestPracticeGroup>
+  <Example
+    height="none"
+    bestPractice={{
+      type: 'do',
+      message:
+        'Zero-pad both date and timestamps for easier comparison across rows.',
+    }}
+  >
+    <ZeroPaddingExample />
+  </Example>
+  <Example
+    height="none"
+    bestPractice={{
+      type: 'dont',
+      message: `Don't omit the leading zero for the date or time. This causes issues with alignment and layout which prevents users from being able to quickly scan and compare the information.`,
+    }}
+  >
+    <ZeroPaddingExample bestPractice={false} />
+  </Example>
+</BestPracticeGroup>
+
+## Examples
+
+### Notifications
+
+When displaying future events such as upcoming updates, expiration dates, and ongoing processes:
+
+- Use the friendlier, more conversational [relative time]() phrasing, such as “in x days”.
+- Provide the date, exact time, and timezone alongside the relative time to remove any ambiguity.
+
+<BestPracticeGroup columns={['auto']}>
+  <Example
+    height="none"
+    bestPractice={{
+      type: 'do',
+      message:
+        'Accompany the relative time with the exact date, time, and timezone in which the event will occur.',
```

Comment:
Should the width of the content here expand to the edge of the notification container or should the notification container's width be shortened?

Reply:
I think that's coming from the theme which causes the notification message to wrap. Let me connect with @halocline on it.